test/test_allocation_domain.cpp

```diff
@@ -303,4 +303,306 @@ TEST_F(AllocationDomainTest, Tensor3d_To_NHWC4d_FwdBwd_CUDA) {
       __FILE__);
 }
 
+// A global->global copy kernel where both inputs are NHWC memory format
```

Comment:
I think you mean `both input and output`

Reply:
Fixed, thanks for catching!